### PR TITLE
docs(app-basics): text input and textarea

### PR DESCRIPTION
**Pull request by Vheissu**
Added in documentation for text input elements as well as textarea elements.

### CODE REVIEW

File: docs/user-docs/app-basics/working-with-form-elements.md [hunk 1]
```diff
@@ -13,7 +13,35 @@
...
+  <input type="text" value.bind="userValue" />
+</form>
+```
+
+You can even bind to other attributes on form elements such as the `placeholder` attribute.
+
+```HTML
+<form>
+  <label>User value</label><br>
+  <input type="text" value.bind="userValue" placeholder"myPlaceholder" />
```
Comment (EisenbergEffect) [inline docs/user-docs/app-basics/working-with-form-elements.md:32]:
```suggestion
  <input type="text" value.bind="userValue" placeholder.bind"myPlaceholder" />
```

**Comment by AureliaEffect** [comment]
JS Framework Benchmark: https://47914-121594173-gh.circle-artifacts.com/0/test/js-framework-benchmark/webdriver-ts-results/table.html

File: docs/user-docs/app-basics/working-with-form-elements.md [hunk 2]
```diff
@@ -13,7 +13,35 @@
...
+  <input type="text" value.bind="userValue" />
+</form>
+```
+
+You can even bind to other attributes on form elements such as the `placeholder` attribute.
+
+```HTML
+<form>
+  <label>User value</label><br>
+  <input type="text" value.bind="userValue" placeholder.bind"myPlaceholder" />
```
Comment (fkleuver) [inline docs/user-docs/app-basics/working-with-form-elements.md:32]:
```suggestion
  <input type="text" value.bind="userValue" placeholder.bind"=myPlaceholder" />
```

**Comment by AureliaEffect** [comment]
JS Framework Benchmark: https://47928-121594173-gh.circle-artifacts.com/0/test/js-framework-benchmark/webdriver-ts-results/table.html

**Comment by AureliaEffect** [comment]
JS Framework Benchmark: https://47989-121594173-gh.circle-artifacts.com/0/test/js-framework-benchmark/webdriver-ts-results/table.html

**Comment by AureliaEffect** [comment]
JS Framework Benchmark: https://49382-121594173-gh.circle-artifacts.com/0/test/js-framework-benchmark/webdriver-ts-results/table.html

**Comment by AureliaEffect** [comment]
JS Framework Benchmark: https://55964-121594173-gh.circle-artifacts.com/0/test/js-framework-benchmark/webdriver-ts-results/table.html

**Comment by fkleuver** [comment]
@Vheissu Is this still up to date or has it been added via a different PR by now?

**Comment by Vheissu** [comment]
@fkleuver This one is good to close. Other PR's addressed this stuff.

**Pull request closed without merging**